src/types/runtime/gravity/FeaturedLink.ts

```diff
@@ -15,6 +15,7 @@ export const FeaturedLink = Record({
   href: String,
   title: String,
   subtitle: String,
```

Comment:
@dzucconi shouldn't this be `.Or(Null)`? (Same with `title` and `href`. Looks like they are all nullable on gravity's side).

Reply:
Yeah probably — one of the things I was thinking about doing with the generation is treat `""` (empty string) as null when specifying types, since that's how it works in practice in JSX conditional rendering. In this case all of the links that were crawled in staging never had actually `null` values. But they frequently had empty string.

Reply:
Interesting. I wonder if that's an issue in torque (like maybe it passes an empty string if no one enters a value?). I'll update to allow nullability in any case!

Reply:
Oh ha, yeah almost certainly — I see the default is actually nil in Gravity which is surprising since we pass `default: ''` for a ton of things